src/Space.ts

```diff
@@ -111,13 +113,9 @@ class Space extends EventEmitter<{ membersUpdate: SpaceMember[] }> {
       timeoutId: setTimeout(timeoutCallback, this.options.offlineTimeout),
     });
   }
-
   private removeLeaver(leaverIndex) {
-    this.leavers.splice(leaverIndex, 1);
-  }
-
-  private resetLeaver(leaverIndex) {
```

Comment:
why are we removing this method? do we not need to reset the timer any more?

Reply:
I merged `removeLeaver` and `resetLeaver` together as I originally planned on emitting the leave event from there, and those methods were only ever called together so could be merged. I can un-merge these since I no longer emit the leave event from there, but I think it is simpler for them to be a single function.